soh/soh/Enhancements/debugconsole.cpp

```diff
@@ -432,6 +446,439 @@ static bool StateSlotSelectHandler(std::shared_ptr<Ship::Console> Console, const
     return CMD_SUCCESS;
 }
 
+static bool InvisibleHandler(std::shared_ptr<Ship::Console> Console, const std::vector<std::string>& args) {
+    if (args.size() != 2) {
+        SohImGui::GetConsole()->SendErrorMessage("[SOH] Unexpected arguments passed");
+        return CMD_FAILED;
+    }
+
+    try {
+        invisibleLink = std::stoi(args[1], nullptr, 10) == 0 ? 0 : 1;
+        if (!invisibleLink) {
+            Player* player = GET_PLAYER(gGlobalCtx);
+            player->actor.shape.shadowDraw = ActorShadow_DrawFeet;
+        }
+
+        return CMD_SUCCESS;
+    } catch (std::invalid_argument const& ex) {
+        SohImGui::GetConsole()->SendErrorMessage("[SOH] Invisible value must be a number.");
+        return CMD_FAILED;
+    }
+}
+
+static bool GiantLinkHandler(std::shared_ptr<Ship::Console> Console, const std::vector<std::string>& args) {
+    if (args.size() != 2) {
+        SohImGui::GetConsole()->SendErrorMessage("[SOH] Unexpected arguments passed");
+        return CMD_FAILED;
+    }
+
+    try {
+        giantLink = std::stoi(args[1], nullptr, 10) == 0 ? 0 : 1;
+        if (giantLink) {
+            paperLink = 0;
+            minishLink = 0;
+        } else {
+            resetLinkScale = 1;
+        }
+
+        return CMD_SUCCESS;
+    } catch (std::invalid_argument const& ex) {
+        SohImGui::GetConsole()->SendErrorMessage("[SOH] Giant value must be a number.");
+        return CMD_FAILED;
+    }
+}
+
+static bool MinishLinkHandler(std::shared_ptr<Ship::Console> Console, const std::vector<std::string>& args) {
+    if (args.size() != 2) {
+        SohImGui::GetConsole()->SendErrorMessage("[SOH] Unexpected arguments passed");
+        return CMD_FAILED;
+    }
+
+    try {
+        minishLink = std::stoi(args[1], nullptr, 10) == 0 ? 0 : 1;
+        if (minishLink) {
+            paperLink = 0;
+            giantLink = 0;
+        } else {
+            resetLinkScale = 1;
+        }
+
+        return CMD_SUCCESS;
+    } catch (std::invalid_argument const& ex) {
+        SohImGui::GetConsole()->SendErrorMessage("[SOH] Minish value must be a number.");
+        return CMD_FAILED;
+    }
+}
+
+static bool AddHeartContainerHandler(std::shared_ptr<Ship::Console> Console, const std::vector<std::string>& args) {
+    if (gSaveContext.healthCapacity >= 0x140)
+        return CMD_FAILED;
```

Comment:
Code style nitpick for here and a few other places, I think brackets or same-line conditions are preferred around the codebase
```suggestion
    if (gSaveContext.healthCapacity >= 0x140) return CMD_FAILED;
```

```suggestion
    if (gSaveContext.healthCapacity >= 0x140) {
      return CMD_FAILED;
    }
```

Reply:
Fixed